src/main/java/com/mroxny/mtba/ScreeningsController.java

```diff
@@ -0,0 +1,44 @@
+package com.mroxny.mtba;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+
+@RestController
+public class ScreeningsController {
+
+    private MultiplexManager multiplexManager;
+
+    @Autowired
+    public ScreeningsController(MultiplexManager multiplexManager) {
+        this.multiplexManager = multiplexManager;
+    }
+
+    @GetMapping("/allScreenings")
+    public ResponseEntity<List<Screening>> getAllScreenings(){
+        List<Screening> screenings = multiplexManager.listAllScreenings();
+        return ResponseEntity.ok(screenings);
+    }
+
+    @GetMapping("/screenings")
+    public ResponseEntity<List<Screening>> getScreenings(@RequestParam LocalDate day, @RequestParam LocalTime startTime, @RequestParam LocalTime endTime){
```

Comment:
Taki zestaw parametrów zakłada, że klient szuka seansów w konkretnym dniu. Czy myślałeś nad użyciem typu LocalDateTime?

Reply:
To już po prostu kwestia mojego założenia że kino jest czynne w danym dniu w przystępnych godzinach (np. 8:00 - 21:00) i filmy nie zaczynają się np. o 23:00 jednego dnia i kończą o 1:00 drugiego. Jak najbardziej można tutaj użyć LocalDateTime ale to już chyba raczej kwestia założeń. Niemniej użyłem tego typu zmiennej w innych częściach dostarczonego rozwiązania